daphne_server/src/roles/aggregator.rs

```diff
@@ -144,18 +144,22 @@ impl DapAggregator<DaphneAuth> for crate::App {
         task_config: &DapTaskConfig,
         req: &DapRequest<DaphneAuth>,
     ) -> Result<Option<String>, DapError> {
+        let unauth_reason = |reason: String| {
+            self.metrics().unauthorized_request(&reason);
+            Ok(Some(reason))
+        };
```

Comment:
Alternatively we could wrap this entire function in another function that increments the metric when we have `Ok(Some(x))`. The goal is to avoid having to increment the metric at each callsite.